lib/routes/30secondsofcode/new-and-popular.ts

```diff
@@ -0,0 +1,41 @@
+import { Data, Route } from '@/types';
+import { load } from 'cheerio';
+import { processList, rootUrl } from './utils';
+import ofetch from '@/utils/ofetch';
+
+export const route: Route = {
+    path: '/',
```

Comment:
https://github.com/DIYgod/RSSHub/blob/4f6501c6002b5575980a8f6c0130dfffa85847be/lib/routes/30secondsofcode/category.ts#L6 will be prioritised instead